deploy/charts/wildside/templates/secret.yaml

```
@@ -0,0 +1,11 @@
+{{- if .Values.secretEnv }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Chart.Name }}-secrets
+type: Opaque
+stringData:
+  {{- range $k, $v := .Values.secretEnv }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)